autosubmitconfigparser/config/basicconfig.py

```diff
@@ -75,19 +76,19 @@ def props(self):
     def expid_dir(exp_id):
         if not isinstance(exp_id, str) or len(exp_id) != 4 or "/" in exp_id:
             raise TypeError("Experiment ID must be a string of 4 characters without the folder separator symbol")
-        return os.path.join(BasicConfig.LOCAL_ROOT_DIR, exp_id)
+        return Path(BasicConfig.LOCAL_ROOT_DIR).joinpath(exp_id)
```

Comment:
I think `joinpath` is more useful when you need to pass that function around. Here, I **think* we could just `Path(BasicConfig.LOCAL_ROOT_DIR, exp_id)`?

Reply:
The only issue is if `exp_id` starts with a slash, but that's true too for joinpath or slash operator.

Reply:
Hi! Addressed this and now we're using `Path()` only. About the slash, an error is raised if the expid contains the `/` character - see the line just above - :)